plugins/main/public/components/common/welcome/agents-welcome.js

```diff
@@ -470,84 +562,154 @@ class AgentsWelcome extends Component {
         />)
     }
 
-    return (
-      <div className="wz-module wz-module-welcome">
-        <div className='wz-module-header-agent wz-module-header-agent-wrapper'>
-          <div className='wz-module-header-agent-main'>
-            {title}
-          </div>
-        </div>
-        <div className='wz-module-agent-body wz-module-agents-padding-responsive'>
-          <EuiPage>
-            <EuiPageBody component="div">
+    renderSCALastScan() {
+      return (
+        <EuiFlexGroup direction='column'>
+          <ScaScan switchTab={this.props.switchTab} {...this.props} />
+        </EuiFlexGroup>
+      );
+    }
 
-              <div className='wz-module-header-nav'>
-                <div>
-                  <EuiPanel grow paddingSize='s' className="wz-welcome-page-agent-info">
-                    <AgentInfo agent={this.props.agent} isCondensed={false} hideActions={true} {...this.props}></AgentInfo>
-                  </EuiPanel>
+    render() {
```

Comment:
**nitpick**: this method is duplicated.

Reply:
Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.